packages/next/build/webpack/loaders/next-flight-server-loader.ts

```diff
@@ -5,17 +5,19 @@ import { parse } from '../../swc'
 import { getBaseSWCOptions } from '../../swc/options'
 import { getRawPageExtensions } from '../../utils'
 
-function isClientComponent(importSource: string, pageExtensions: string[]) {
-  return new RegExp(`\\.client(\\.(${pageExtensions.join('|')}))?`).test(
-    importSource
-  )
-}
+const getIsClientComponent =
```

Comment:
I dont know if it's a common usage for `getIsXxx`, how about `createClientComponentFilter` or sth similar?

Reply:
It's a very common naming convention, for example: https://cs.github.com/?scopeName=All+repos&scope=&q=%2FgetIs%5Cw%2B%5C%28%2F

Reply:
They're likely returning the value instead of creating another getter function, like `getIsSsr(): boolean`. Just wanna flag it as a creator of getters, not very blocking